components/git/vote.js

```diff
@@ -0,0 +1,84 @@
+import auth from '../../lib/auth.js';
+import { parsePRFromURL } from '../../lib/links.js';
+import CLI from '../../lib/cli.js';
+import Request from '../../lib/request.js';
+import { runPromise } from '../../lib/run.js';
+import VotingSession from '../../lib/voting_session.js';
+
+export const command = 'vote [prid|options]';
+export const describe =
+  'Manage the current landing session or start a new one for a pull request';
+
+const voteOptions = {
+  abstain: {
+    type: 'boolean',
+    default: false,
+    describe: 'Abstain from the vote.'
+  },
+  'decrypt-key-part': {
+    describe: 'Publish a key part as a comment to the vote PR.',
+    default: false,
+    type: 'boolean'
+  },
+  'gpg-sign': {
+    describe: 'GPG-sign commits, will be passed to the git process',
+    alias: 'S'
+  },
+  'post-comment': {
+    describe: 'Post the comment on GitHub on the behalf of the user',
+    default: false,
+    type: 'boolean'
+  },
+  protocol: {
+    describe: 'The protocol to use to clone the vote repository and push the eventual vote commit',
+    type: 'string'
+  }
+};
+
+let yargsInstance;
+
+export function builder(yargs) {
+  yargsInstance = yargs;
+  return yargs
+    .options(voteOptions)
+    .positional('prid', {
```

Comment:
should this be a number, like in `git node land`?

Reply:
It can be a number, however I don't expect much folks to use it as such (because there won't be any vote PR targeted to nodejs/node)

Reply:
Ok, that makes sense